kitsune/users/views.py

```diff
@@ -187,15 +187,51 @@ def deactivation_log(request):
     return render(request, "users/deactivation_log.html", {"deactivations": deactivations})
 
 
+def questions(request, username):
+    # plus sign (+) is converted to space
+    username = username.replace(" ", "+")
+    profile = get_object_or_404(Profile, user__username=username, user__is_active=True)
+
+    questions = paginate(request, profile.user.questions.order_by("-created"))
+
+    return render(
+        request,
+        "users/questions.html",
+        {
+            "profile": profile,
+            "questions": questions,
+        },
+    )
+
+
+def answers(request, username):
+    # plus sign (+) is converted to space
+    username = username.replace(" ", "+")
+    profile = get_object_or_404(Profile, user__username=username, user__is_active=True)
+
+    # don't paginate this, we have contributors with an incredible number of questions answered
+    # which means the later pages would create queries taking > 10 seconds to run
+    answers = profile.user.answers.order_by("-created").select_related("question")[:100]
+
+    return render(
+        request,
+        "users/answers.html",
+        {
+            "profile": profile,
+            "answers": answers,
+        },
+    )
+
+
 @require_GET
-def documents_contributed(request, username):
+def documents(request, username):
```

Comment:
imo the previous name was better. `documents` is too generic and it's not clear where is referring to

Reply:
It's inconsistently named with its url which is `/user/:username/documents`, I also can't think of anything else which `kitsune.users.views.{ documents, questions, answers }` could be referring to, other than the instances of those which the user authored.

Reply:
True but you also need the context of the project and/or the `users` app. In this case consistently naming all of these endpoints like `{documents, questions, answers}_contributed` should make things clearer. I would go with explicit is better than implicit